webapp/app/forms/flows/lotse_flow.py

```diff
@@ -354,7 +358,9 @@ def _collect_data_for_step(self, step, form_data, missing_fields=None):
                 field = getattr(step.form, attr)
                 label, value = self._generate_value_representation(field, form_data[attr])
                 if value:
-                    step_data[label] = value
+                    step_value = step.get_overview_value_representation(value)
+                    if step_value is not None:
+                        step_data[label] = step_value
```

Comment:
If the function get_overview_value_representation() returns `None`, the value will not be displayed.

Reply:
Maybe you could add this as a comment in the code?

Reply:
I think we should again think more about how we combine this part with the `_generate_value_representation`. However, this could also be a task for the refactoring of the steps

Reply:
On main we already had a ToDo comment for combining `_generate_value_representation`. I added the comment